components/apimgt/org.wso2.carbon.apimgt.keymgt/src/main/java/org/wso2/carbon/apimgt/keymgt/handlers/DefaultKeyValidationHandler.java

```diff
@@ -158,10 +158,17 @@ public boolean validateScopes(TokenValidationContext validationContext) throws A
         } else {
             resourceArray = new ArrayList<>(Arrays.asList(resourceList));
         }
+
+        String actualVersion = validationContext.getVersion();
```

Comment:
it's wrong to call this actual version right?

Reply:
Let's make it as apiVersion and then assign the actual version by splitting.